src/functions/SetupTeardown.ps1

```diff
@@ -258,14 +249,8 @@ function AfterAll {
         This example uses AfterAll to clean up a sample-file generated only for
         the tests in the Describe-block.
 
-    .LINK
-        https://pester.dev/docs/commands/AfterAll
-
```

Comment:
This was on purpose. It enables `Get-Help AfterAll -Online` to open the online version.
Supposed to be there for every function, however that might be done yet. Just updated old wiki-links on last edit.

Reply:
Allright, I can reverse if need be. Before I do... is this currently consistent behavior? Simply put: does every command have a link like this one pointing to the pester website ?

Reply:
Thanks, I see. I will undo this change so the link-to-self comes back in.

Reply:
Pretty sure it's missing multiple places, but I intended to add them + a help-test requiring it when pester.dev was updated (which I didn't have an ETA for) 👍

Reply:
Just added it back. The pester.dev website should be updated shorty, is awaiting a merge. https://github.com/pester/docs/pull/78